ios/CameraView+TakePhoto.swift

```diff
@@ -47,6 +47,15 @@ extension CameraView {
         }
         if photoOutput.availablePhotoCodecTypes.contains(photoCodec) {
           format = [AVVideoCodecKey: photoCodec]
+          if (photoCodec == AVVideoCodecType.jpeg) {
+            if let jpegQuality = options["jpegQuality"] as? Float {
+              guard jpegQuality >= 0.0 && jpegQuality <= 1.0 else {
+                promise.reject(error: .parameter(.invalid(unionName: "JpegQuality", receivedValue: jpegQuality)))
+                return
+              }
+              format![AVVideoQualityKey] = jpegQuality
```

Comment:
Does that actually work? Will it have lower quality when you pass something in in the resulting photo? Seems weird that they don't have a nicer API for this, no?

Reply:
yes, it works !

there's another api [jpegData(compressionQuality:)](https://developer.apple.com/documentation/uikit/uiimage/1624115-jpegdata), but you need to have an image already, something that we don't have at this step of the flow

Reply:
Oh wow I did not know that - I think then we have a way to do it on iOS as well!

Reply:
Oh well actually yea I don't really like that this has to be configured with the session instead of just `takePhoto` options... We need to find another solution that works like on Android

Reply:
yes, it's quirky :(

not sure there's another solution, although i haven't looked at it in quite a while

Reply:
Yea I think if we convert to CGImage first then we can compress with quality, but imo it's an unnecessary round-trip.

I hope there's gonna be an API for that soon, otherwise that's just how it's gonna be 🤷‍♂️